toolkit/Makefile

```diff
@@ -82,6 +82,7 @@ STATUS_FLAGS_DIR   ?= $(BUILD_DIR)/make_status
 CHROOT_DIR         ?= $(BUILD_DIR)/worker/chroot
 IMAGEGEN_DIR       ?= $(BUILD_DIR)/imagegen
 TOOLCHAIN_RPMS_DIR ?= $(BUILD_DIR)/toolchain_rpms
+TIMESTAMP_DIR      ?= $(BUILD_DIR)/timestamp
```

Comment:
This is super timely.

One interesting area as per my investigation is `lock-contention` w.r.t to our `logger`.
We should probably time a full build with `LOG_LEVEL=info` & `LOG_LEVEL=panic`
The above would give us a good pointer for further investigation.

Reply:
I think we might need to tinker with the logging library a bit first. My recollection is that we always log at minimum `debug` level to the log files regardless of what the console output level is.